src/Datadog.Trace.ClrProfiler.Native/rejit_handler.cpp

```diff
@@ -168,7 +249,26 @@ void RejitHandler::EnqueueThreadLoop(RejitHandler* handler)
             break;
         }
 
-        hr = profilerInfo->RequestReJIT((ULONG) item->m_length, item->m_modulesId.get(), item->m_methodDefs.get());
+        if (profilerInfo10 != nullptr)
+        {
+            // RequestReJITWithInliners is currently always failing...
+            // more research is required, meanwhile we fallback to the normal RequestReJIT and manual track of inliners.
```

Comment:
If might be good to send an email to David Mason to ask any questions about the API and if there are any results that look wrong